include/exadg/convection_diffusion/preconditioners/multigrid_preconditioner.cpp

```diff
@@ -326,7 +327,7 @@ void
 MultigridPreconditioner<dim, Number>::update_smoothers()
 {
   // Skip coarsest level
-  for(unsigned int level = this->coarse_level + 1; level <= this->fine_level; ++level)
+  for(unsigned int level = 0; level <= this->get_number_of_levels() - 1; ++level)
```

Comment:
Here is the mistake causing the failing test described here https://github.com/exadg/exadg/pull/448/files#r1205088650

Reply:
fixed here https://github.com/exadg/exadg/pull/448/commits/b0d63fffe82ddc9dff2a54a1d4861b88db2ecbe5